distributions/test/src/test/java/com/connexta/ion/replication/distributions/test/ReplicationITest.java

```diff
@@ -0,0 +1,144 @@
+package com.connexta.ion.replication.distributions.test;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.collect.ImmutableMap;
+import com.jayway.restassured.path.xml.XmlPath;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+// TODO: why does this need to end in "test" for the tests to run?
+public class ReplicationITest extends IntegrationTest {
```

Comment:
I think we should be able to give the plugin a regex to tell it which files are itests.

Reply:
I looked into it more, apparently we use the fail safe plugin for itests. I Started using that and named the file ReplicationIT. I don't think it gives us any benefit at the moment however, because we're not doing anything in the pre/post-integration-tests phases of the plugin. Testcontainers handles any setup or tear down of the test container instead.